test/functional/inspec_vendor_test.rb

```diff
@@ -143,4 +143,49 @@
       vendor_files.must_equal vendor_cache_files
     end
   end
+
+  it 'vendors profiles when using a local path' do
+    local_depends_path = File.join(profile_path, 'local-depends')
```

Comment:
I'm not clear on what "Vendors profiles that are referenced locally (e.g. `path://`)" means.  I looked at #1974, which is a desire to have a git-referenced profile able to use a `path:` inspec.yml option to find the profile deeper within the git repo.  This test does not cover that, nor do I see anywhere that the URI schema 'path://' is used.  Could you clarify?

Reply:
Hmm...I read it as wanting the ability to specify a local directory relative to the profile.

Granted, I can see that the original post wanted to use `path:` with `url:` which is something we do not support.

I'll update that issue and ask if what we have is sufficient, if not I'll suggest we make it a feature request.